prosody/Dockerfile

```diff
@@ -1,10 +1,25 @@
 FROM jitsi/base
 
+ADD https://packages.prosody.im/debian/pool/main/p/prosody/prosody_0.11.1-1~stretch1_amd64.deb /tmp/prosody.deb
```

Comment:
Please add the repo instead of pulling the file manually, this way we'll get automatic updates as they are pushed to the repo.

Reply:
Ohh yeah you're right. I changed it.